lib/liquid/combinators/tags/include.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Liquid.Combinators.Tags.Include do
+  @moduledoc """
+  Include allows templates to relate with other templates
```

Comment:
"Include enables the possibility to include and render other liquid templates. Templates can also be recursively included"